sdk/url_builders.go

```diff
@@ -0,0 +1,63 @@
+package sdk
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+const (
+	SystemSubDomain = "system"
+)
+
+func FormatEndpointURL(gatewayURL string, event *Event) (string, error) {
```

Comment:
Hi @martindekov I see duplication here.. can we do the SDK PR first then re-vendor rather than having a much bigger PR with duplication?